src/Repository.php

```diff
@@ -553,6 +553,18 @@ public function notActive($name)
     }
 
     /**
+     * Determine if the given module exist and is activated.
+     *
+     * @param string $name
+     *
+     * @return bool
+     */
+    public function isActive($name)
+    {
+        return ($module = $this->find($name)) && $module->active();
```

Comment:
It will throw a `ModuleNotFoundException` exception, which you can catch to build your menu.